Packs/NistNVD/Integrations/NistNVD/NistNVD.py

```diff
@@ -0,0 +1,322 @@
+                pretty_dict['CVSSv3 Vector String'] = i['impact']['baseMetricV3']['cvssV3']['vectorString']
+                pretty_dict['CVSSv3 Attack Vector'] = i['impact']['baseMetricV3']['cvssV3']['attackVector']
+                pretty_dict['CVSSv3 Attack Complexity'] = i['impact']['baseMetricV3']['cvssV3']['attackComplexity']
+                pretty_dict['CVSSv3 Privileges Required'] = i['impact']['baseMetricV3']['cvssV3']['privilegesRequired']
+                pretty_dict['CVSSv3 User Interaction'] = i['impact']['baseMetricV3']['cvssV3']['userInteraction']
+                pretty_dict['CVSSv3 Scope'] = i['impact']['baseMetricV3']['cvssV3']['scope']
+                pretty_dict['CVSSv3 Confidentiality Impact'] = i['impact']['baseMetricV3']['cvssV3']['confidentialityImpact']
+                pretty_dict['CVSSv3 Integrity Impact'] = i['impact']['baseMetricV3']['cvssV3']['integrityImpact']
+                pretty_dict['CVSSv3 Availability Impact'] = i['impact']['baseMetricV3']['cvssV3']['availabilityImpact']
+            """if (('configurations') in i):
+                cpe23UriString = ''
+                for l in i['configurations']['nodes']:
+                    if(('children') in l):
+                        for m in l['children']:
+                            cpe23Uri = []
+                            for n in m['cpe_match']:
+                                cpe23Uri += n['cpe23Uri']+ '\n' #+ n['vulnerable']
+                                #pretty_dict['vulnerable'] = n['vulnerable']
+                                #pretty_dict['cpe23Uri'] = n['cpe23Uri']
+                        pretty_dict['configurations'] = cpe23UriString
+                    elif(('cpe_match') in l):
+                        for m in l['cpe_match']:
+                            cpe23UriString += m['cpe23Uri'] + '\n' #+ m['vulnerable']
+                            #pretty_dict['vulnerable'] = m['vulnerable']
+                            #pretty_dict['cpe23Uri'] = m['cpe23Uri']
+                        pretty_dict['configurations'] = cpe23UriString"""
+            pretty_list.append(pretty_dict)
+    elif ('vulns') in req:
+        if (not len(req['vulns'])):
+            demisto.results("Vendor name may be wrong or no CPE added")
+
+    if (('result') in req):
+        if (not len(req['result']['CVE_Items'])):
+            demisto.results("There were no vulnerability in the criteria you were looking for.")
+    return pretty_list
+
+
+def generalSearch():
+
+    base_url = urljoin(demisto.params()['url'], '/rest/json/cves/1.0')
+    time = int(demisto.args().get('time'))
+    last_time = datetime.today() - timedelta(hours=int(time))
+    start_date = last_time.strftime('%Y-%m-%dT%H:%M:%S:000')
+    startIndex = demisto.args().get('startIndex')
+    resultsPerPage = demisto.args().get('resultsPerPage')
+    additional_parameters = '?modStartDate=' + start_date + ' UTC-05:00' + \
+        '&startIndex=' + str(startIndex) + '&resultsPerPage=' + str(resultsPerPage)
```

Comment:
Is it ok that `startIndex` and `resultPerPage` will be `None`? since they are not required

Reply:
I used these parameters for paging and if they are None,  no result is returned and these parameters have default initialize values.

Reply:
They do have default values, but they can be overridden by the user.
Maybe you should verify its content before?